apps/server/src/app/fhir.controller.ts

```diff
@@ -693,7 +693,9 @@ export class FhirController extends BaseController {
     @Query('name') name?: string,
     @Query('title') title?: string,
     @Query('_content') resourceContent?: string,
-    @Query('type') structureDefinitionType?: string) {
+    @Query('type') structureDefinitionType?: string,
+    @Query('url') structureDefinitionUrl?: string,
+    @Query('isLightweight') isLightweight?: boolean) {
```

Comment:
I think you can do `@Query('isLightweight') isLightweight = true` to set it as a default, and then later you can always check for `if (isLightweight) { ... }`

Reply:
ah! good catch, thanks